### ISSUE TITLE
Include tanzu-to-go workshop content in end-to-end automated tests

### ISSUE THREAD

**Issue by joshrosso** [vmware-tanzu/community-edition#2903]
## Feature Request

The solution engineering team is running a workshop that includes TCE. It exists at [https://github.com/Tanzu-Solutions-Engineering/tanzu-to-go-workshop](https://github.com/Tanzu-Solutions-Engineering/tanzu-to-go-workshop).

I've not personally dug into this repo, but bringing up the suggestion to consider whether there's a good hook for us to test our releases against this work/process.

So in summary, first step would be to look at the project and see if there's value in partnering here.

**Comment by karuppiah7890** [comment]
Thanks for bringing this up @joshrosso ! We'll explore the https://github.com/Tanzu-Solutions-Engineering/tanzu-to-go-workshop repo 😄
